addons/l10n_cl/data/l10n_cl_chart_data.xml

```diff
@@ -163,7 +163,7 @@
     <record id="account_11320" model="account.account.template">
         <field name="code">110320</field>
         <field name="name">Anticipo Proveedores</field>
-        <field ref="account.data_account_type_current_assets" name="user_type_id"/>
+        <field ref="account.data_account_type_receivable" name="user_type_id"/>
```

Comment:
?

Reply:
@jco-odoo This is because the advance (anticipo) in Chile is used that way. For example if you pay to a vendor something in advance and you make the payment using the account "proveedores" (which is a payable account). You've got in the balance report, a liability account with a figure in debit column. This is always bad opinionated by all accountants I know. This way (using it as receivable) is relatively easy with 16 and 17 reconciliation features to apply the advance amount to an invoice.

Reply:
@jco-odoo just to add on @Danisan comment we have the account "Anticipo Clientes" as a payable by default, if we want to be consistent I would either change the Anticipo Proveedores to receivable or the Anticipo Clientes to a current liability... If we change them to current assets or liabilities (the international standard) users will have to change them manually if they want to.

Reply:
Hey @jco-odoo let us know what you think so we can move forward. Cheers!

Reply:
Should not it be an expense? (most accounts do like that)  And we already have a receivable line on an invoice, so it could mess up the accounting entries.  (It would be good to check how it is actually supposed to be used)

Reply:
I explained this to @fvz-odoo already. It is not an expense, simply because this is going to be recovered later, when the vendor delivers the products or services, and invoices them.

Reply:
@Danisan That is no reason to make it receivable/payable.  The best would be to see how you want it to work.

Reply:
Following up on this I think we need @chklop opinion. 

Context: @Danisan is changing the type of the account for the advanced payments to vendors from current assets to accounts receivable. 2 things here, a) we have the account for the advanced payments from customers as accounts payable by default, and b) all accountants in Chile use them as accounts receivable (also in other latam countries like Mexico and Colombia for example).

Options: 
1. Accept Dani"s change as it is the common practice in Chile. Pro: Makes sense for the localization and everybody in Chile uses it like that. Con: It does not comply with international standards (which I don't think it is wrong as we're talking about the Chilean loc here).
2. Change the type of account for the advanced payments from customers to a liability to be consistent with the international standards. Pro: Consistency within accounts in Odoo and follow international standards. Con: All Chilean customers (and other localizations) will need to change the type manually to what they're use to. 

Bonus: if we have it as assets (or liability) you cannot reconcile it with an invoice, I've had this issue with a Mexican customer.

Since this is not that blocking and there's a "workaround" we should not spend that much time on this type of things IMO. Still I think that if in many locs it is used in a certain way, we can make some exceptions and not follow international standards. But I leave it to you.

Cheers!

Fernando 

@jco-odoo @ren-odoo @masi-odoo @vbe-odoo

Reply:
I'm not against it, in practice this is heavily dependant on local accountants preference and accounting wise there are multiple different situations encountered with Advance Payments (whether or not they are linked to a running contract, whether or not they are linked to an asset, etc). In Belgium, our Trade Debtors Advance Payments accounts are already of type Account Receivable. 
![image](https://github.com/odoo/odoo/assets/93644924/bfd6fb76-6227-409e-a047-57235c118670)

Reply:
Thanks for your answer @chklop, we talked about it this morning with @jco-odoo, he said he would talk to you about it. Let us know the final decision, 😉.

Cheers!

Fernando

@masi-odoo @ren-odoo